lib/page/completion/completion_page.dart

```diff
@@ -1,10 +1,76 @@
+import 'dart:async';
+
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:lottie/lottie.dart';
+import 'package:survey/constants.dart';
+import 'package:survey/di/di.dart';
+import 'package:survey/gen/assets.gen.dart';
+import 'package:survey/gen/colors.gen.dart';
+import 'package:survey/navigator.dart';
+import 'package:survey/resource/dimens.dart';
 
-class CompletionPage extends StatelessWidget {
+class CompletionPage extends StatefulWidget {
   const CompletionPage({super.key});
 
+  @override
+  State<CompletionPage> createState() => _CompletionPageState();
+}
+
+class _CompletionPageState extends State<CompletionPage>
+    with TickerProviderStateMixin {
+  final _appNavigator = getIt.get<AppNavigator>();
+
+  late final AnimationController _animationController =
+      AnimationController(vsync: this)
+        ..addStatusListener((status) {
+          if (status == AnimationStatus.completed) {
+            // Wait for 2 seconds after animation is completed before navigating back to the Home screen
+            Timer(Duration(seconds: Constants.snackBarDurationInSecond), () {
```

Comment:
Why do we use `snackBarDurationInSecond` for this completion page's delay time? I know it's 2 secs, but it does not make sense.